### PR TITLE
[Excel] (performance) suspendScreenUpdatingUntilNextSync method

### PR DESCRIPTION
**Pull request by AlexJerabek**
This adds a section about `Application.suspendScreenUpdatingUntilNextSync()`. I also did minor organizational changes to the Performance article.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/performance.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/excel/performance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812070143298608-605%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YCUvuMGLYFc1ND10wIeJ6pu9vxCmqNI5FQNJTmRYOxI%3D&st=2018-12-07T01%3A43%3A18Z&se=2019-01-07T01%3A48%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/performance.md [hunk 1]
```diff
@@ -71,11 +71,15 @@ _Where:_
 
 Please be aware that some of the “properties” under an object may have the same name as another object. For example, `format` is a property under range object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()`, which is an empty load() call that can cause performance problems as outlined previously. To avoid this, your code should only load the “leaf nodes” in an object tree. 
 
-## Suspend calculation temporarily
+## Suspend Excel processes temporarily
 
-If you are trying to perform an operation on a large number of cells (for example, setting the value of a huge range object) and you don't mind suspending the calculation in Excel temporarily while your operation finishes, we recommend that you suspend calculation until the next ```context.sync()``` is called.
+When your add-in edits an Excel workbook through the JavaScript API, the experience is similar to when a user edits it through the UI. This can be slower than desired when dealing with large data sets, due to the side processes Excel runs. Some of these can be controlled to yield a performance benefit.
```
Comment (Rick-Kirkham) [inline docs/excel/performance.md]: I'm not convinced that the "experience is similar". Even if it is, do we need to say this?

File: docs/excel/performance.md [hunk 2]
```diff
@@ -71,11 +71,15 @@ _Where:_
 
 Please be aware that some of the “properties” under an object may have the same name as another object. For example, `format` is a property under range object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()`, which is an empty load() call that can cause performance problems as outlined previously. To avoid this, your code should only load the “leaf nodes” in an object tree. 
 
-## Suspend calculation temporarily
+## Suspend Excel processes temporarily
```
Comment (kbrandl) [inline docs/excel/performance.md]: Would this read better as "Temporarily suspend Excel processes" ? Just a suggestion; take it or leave it :)

File: docs/excel/performance.md [hunk 3]
```diff
@@ -116,6 +120,18 @@ Excel.run(async function(ctx) {
 })
 ```
 
+### Suspend screen updating
+
+> [!NOTE]
+> The `suspendScreenUpdatingUntilNextSync()` method described in this article requires the beta version of the Office JavaScript library from [Office.js CDN](https://appsforoffice.microsoft.com/lib/beta/hosted/office.js). The [type definition file] (https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts) is also found at the CDN. For more information on our upcoming APIs, please visit the [open spec](https://github.com/OfficeDev/office-js-docs/tree/ExcelJs_OpenSpec) on GitHub.
+
+Excel displays changes your add-in makes approximately as they happen in the code. For large, iterative data sets, you may not need to see this progress on the screen. `Application.suspendScreenUpdatingUntilNextSync()` pauses visual updates to Excel until the ad-in calls `context.sync()` (or the `Excel.run` call ends, implictly calling `context.sync`). Be aware, Excel will not show any signs of activity until the next sync. Your add-in should either give users guidance to prepare them for this delay or provide a status bar to demonstrate activity.
```
Comment (kbrandl) [inline docs/excel/performance.md]: - typo: "ad-in" -> "add-in"
- typo: "implictly" -> "implicitly"

Comment (kbrandl) [inline docs/excel/performance.md]: consider changing "...you may not need to see this progress on the screen." -> "...you may not need to see this progress on the screen in real-time."

Comment (kbrandl) [inline docs/excel/performance.md]: "...pauses visual updates to Excel until the ad-in calls `context.sync()` (or the `Excel.run` call ends, implictly calling `context.sync`).
->
"...pauses visual updates to Excel until the add-in calls `context.sync()`, or until the `Excel.run` ends (implicitly calling `context.sync`)."

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/excel/performance.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/excel/performance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812071854435810-605%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3Mp7hXa4AaV2Drfge79a%2FIZQji4KI2mKLFmABaoo6DU%3D&st=2018-12-07T18%3A51%3A18Z&se=2019-01-07T18%3A56%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/excel/performance.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/excel/performance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812071855043997-605%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xRrWX2%2B3Vrugur22I6VqyvwWCsKwfHabljsaIONPsCE%3D&st=2018-12-07T18%3A52%3A38Z&se=2019-01-07T18%3A57%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/performance.md (hunk 2 above)
Comment (AlexJerabek) [reply docs/excel/performance.md]: I agree it sounds better, but I wanted to match the verb-first header patterns.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/excel/performance.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Calc/docs/excel/performance.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812071907040699-605%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2F%2BHVVadv0M93uzcYtCs8wAsWKREStY12UU1WrzmGod0%3D&st=2018-12-07T19%3A03%3A14Z&se=2019-01-07T19%3A08%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**